tests/testthat/test-families.R

```diff
@@ -1,40 +0,0 @@
-context("general family tests")
```

Comment:
Why delete this file without a replacement?

Reply:
I tired to isolate problems on Travis. I will add it back.